db/migrate/20191024094436_create_pois.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePois < ActiveRecord::Migration[5.2]
+  def change
+    create_table :pois do |t|
+      t.references :destination, foreign_key: true
+      t.string :name
+      t.string :type
```

Comment:
I think `type` is a reserved/key word in Graphql. For the `Transportation` table, initially, I had a `type` attribute. When I tried to generate the `Transportation` model in Graphql, there were naming collisions, and I changed my `type` attribute to `mode.` 

If you do run into any naming collisions, the `type` attribute could be renamed to `categories`.